### PR TITLE
Add version to constant API_PREFERRED_VERSIONS

### PR DESCRIPTION
**Pull request by benjaminapetersen**
TODO:
- [x] `selfsubjectrulesreviews` is either `v1` or `v1beta1` (unclear which we are using)
- [ ] ~~pull latest catalog images & get the correct version for `serviceclasses`, `serviceinstancecredentials`, and `serviceinstances`~~ todo post beta

@spadgett

### CODE REVIEW

File: dist/origin-web-common.js [hunk 1]
```diff
@@ -1425,39 +1425,39 @@ angular.module('openshiftCommonUI')
...
-    templates:                        {group: 'template.openshift.io',  resource: 'templates'}
+      appliedclusterresourcequotas:     {group: 'quota.openshift.io',     version: 'v1',      resource: 'appliedclusterresourcequotas' },
+      builds:                           {group: 'build.openshift.io',     version: 'v1',      resource: 'builds' },
+      'builds/clone':                   {group: 'build.openshift.io',     version: 'v1',      resource: 'builds/clone' },
+      'buildconfigs/instantiate':       {group: 'build.openshift.io',     version: 'v1',      resource: 'buildconfigs/instantiate' },
+      buildconfigs:                     {group: 'build.openshift.io',     version: 'v1',      resource: 'buildconfigs' },
+      configmaps:                       'configmaps',
+      deployments:                      {group: 'apps',                   version: 'v1beta1', resource: 'deployments' },
+      deploymentconfigs:                {group: 'apps.openshift.io',      version: 'v1',      resource: 'deploymentconfigs' },
+	    'deploymentconfigs/instantiate':  {group: 'apps.openshift.io',      version: 'v1',      resource: 'deploymentconfigs/instantiate' },
```
Comment (spadgett) [inline dist/origin-web-common.js]: indentation

File: dist/origin-web-common.js [hunk 2]
```diff
@@ -1425,39 +1425,39 @@ angular.module('openshiftCommonUI')
...
+      projects:                         {group: 'project.openshift.io',   version: 'v1',      resource: 'projects'},
+      projectrequests:                  {group: 'project.openshift.io',   version: 'v1',      resource: 'projectrequests'},
+      persistentvolumeclaims:           'persistentvolumeclaims',
+      replicasets:                      {group: 'extensions',             version: 'v1beta1', resource: 'replicasets' },
+      replicationcontrollers:           'replicationcontrollers',
+      resourcequotas:                   'resourcequotas',
+      rolebindings:                     'rolebindings',
+      routes:                           {group: 'route.openshift.io',     version: 'v1',      resource: 'routes' },
+      secrets:                          'secrets',
+      selfsubjectrulesreviews:          {group: 'authorization.k8s.io',   version: 'v1',      resource: 'selfsubjectrulesreviews'},
```
Comment (spadgett) [inline dist/origin-web-common.js]: authorization.openshift.io

File: dist/origin-web-common.js [hunk 3]
```diff
@@ -1425,39 +1425,39 @@ angular.module('openshiftCommonUI')
...
-    serviceinstancecredentials:       {group: 'servicecatalog.k8s.io',  resource: 'serviceinstancecredentials' },
-    serviceinstances:                 {group: 'servicecatalog.k8s.io',  resource: 'serviceinstances' },
-    statefulsets:                     {group: 'apps',                   resource: 'statefulsets' },
-    templates:                        {group: 'template.openshift.io',  resource: 'templates'}
+      appliedclusterresourcequotas:     {group: 'quota.openshift.io',     version: 'v1',      resource: 'appliedclusterresourcequotas' },
+      builds:                           {group: 'build.openshift.io',     version: 'v1',      resource: 'builds' },
+      'builds/clone':                   {group: 'build.openshift.io',     version: 'v1',      resource: 'builds/clone' },
+      'buildconfigs/instantiate':       {group: 'build.openshift.io',     version: 'v1',      resource: 'buildconfigs/instantiate' },
+      buildconfigs:                     {group: 'build.openshift.io',     version: 'v1',      resource: 'buildconfigs' },
+      configmaps:                       'configmaps',
```
Comment (spadgett) [inline dist/origin-web-common.js]: So we add a version to these, too. It just doesn't have a group.

**Comment by spadgett** [comment]
> pull latest catalog images & get the correct version for serviceclasses, serviceinstancecredentials, and serviceinstances

We can wait until it's beta before we add the version

File: dist/origin-web-common.js (hunk 2 above)
Comment (benjaminapetersen) [reply dist/origin-web-common.js]: Ah, great, thx. that clarifies.

**Comment by benjaminapetersen** [comment]
Updated, going to test the non-grouped locally

File: src/constants/apiPreferredVersions.js [hunk 4]
```diff
@@ -2,37 +2,38 @@
...
-    serviceinstancecredentials:       {group: 'servicecatalog.k8s.io',  resource: 'serviceinstancecredentials' },
-    serviceinstances:                 {group: 'servicecatalog.k8s.io',  resource: 'serviceinstances' },
-    statefulsets:                     {group: 'apps',                   resource: 'statefulsets' },
-    templates:                        {group: 'template.openshift.io',  resource: 'templates'}
+      appliedclusterresourcequotas:     {group: 'quota.openshift.io',         version: 'v1',      resource: 'appliedclusterresourcequotas' },
+      builds:                           {group: 'build.openshift.io',         version: 'v1',      resource: 'builds' },
+      'builds/clone':                   {group: 'build.openshift.io',         version: 'v1',      resource: 'builds/clone' },
+      'buildconfigs/instantiate':       {group: 'build.openshift.io',         version: 'v1',      resource: 'buildconfigs/instantiate' },
+      buildconfigs:                     {group: 'build.openshift.io',         version: 'v1',      resource: 'buildconfigs' },
+      configmaps:                       {version: 'v1',                   resource: 'configmaps'},
```
Comment (spadgett) [inline src/constants/apiPreferredVersions.js]: nit: indentation

**Comment by benjaminapetersen** [comment]
@spadgett this one should be good to go.

**Comment by spadgett** [comment]
Closing and reopening to kick travis tests

**Comment by spadgett** [comment]
@benjaminapetersen verify dist failed

**Comment by benjaminapetersen** [comment]
checking

**Pull request merged**